client/web/src/site-admin/permissions-center/PermissionsSyncJobNode.tsx

```diff
@@ -38,14 +38,14 @@ const JOB_REASON_TO_READABLE_REASON: Record<PermissionsSyncJobReason, string> =
     REASON_MANUAL_REPO_SYNC: 'Repository synchronization triggered manually',
     REASON_MANUAL_USER_SYNC: 'User synchronization triggered manually',
     REASON_REPO_NO_PERMS: 'Repository has no permissions',
-    REASON_REPO_OUTDATED_PERMS: 'Repository had outdated permissions',
+    REASON_REPO_OUTDATED_PERMS: 'Regular refresh of repository permissions',
```

Comment:
Can we also change the enum values?
```suggestion
    REASON_REPO_REGULAR_SCHEDULE: 'Regular refresh of repository permissions',
```